7.2.0/cleanup.sh

```
@@ -0,0 +1,381 @@
+
+  # --------------------------------------------------------
+  # Step 2: Clean shared DB resources
+  # --------------------------------------------------------
+  SHARED_CLEANUP_SCRIPT="./${SHARED_DB_TYPE}/shared_db_cleanup.sh"
+  if [[ ! -f "$SHARED_CLEANUP_SCRIPT" ]]; then
+    echo "Error: Shared DB cleanup script for type '$SHARED_DB_TYPE' not found. Exiting."
+    exit 1
+  fi
+
+  for (( i=0; i<${#SHARED_DB_USERS[@]}; i++ )); do
+    SHARED_PARAMS=($(get_db_params "$SHARED_DB_TYPE" "$i" "shared"))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)